dev-resources/puppetlabs/general_puppet/general_puppet_int_test/custom_trusted_oid_mapping.yaml

```diff
@@ -5,4 +5,7 @@ oid_mapping:
     longname: 'Shining Finger'
   1.3.6.1.4.1.34380.1.2.3:
     shortname: 'short'
-    longname: 'A Very Long Name'
\ No newline at end of file
+    longname: 'A Very Long Name'
+  1.3.6.1.4.1.34380.1.3.39:
+    shortname: 'pp_auth_doodad'
+    longname: 'PP Auth Doodad Long Name'
```

Comment:
How is this used in the tests?

Reply:
like, I see the check you're doing below, just curious like.. what the other info is used for and how..

Reply:
The longname isn't used in tests but is part of the custom oid spec: https://puppet.com/docs/puppet/latest/config_file_oid_map.html

Reply:
Does that answer your question?

Reply:
Partly! so the `longname` just needs to be there for Reasons™ 👍 
but what about the oid? Is that something you made up or is that specific to this extension or ???

Reply:
It already existed on the cert (these extensions have to baked into the CSR). It just wasn't being properly displayed before because of the bug this PR fixed.

Reply:
Both long and short names for the oid I made up. We only define names for .1 and .13 https://github.com/puppetlabs/puppet/blob/master/lib/puppet/ssl/oids.rb#L69-L70

Reply:
ahh okay, I think I understand better now, thank you!!